src/Microsoft.PowerApps.TestEngine/Modules/TestEngineExtensionChecker.cs

```diff
@@ -278,6 +290,23 @@ public virtual bool Validate(TestSettingExtensions settings, string file)
         public bool VerifyContainsValidNamespacePowerFxFunctions(TestSettingExtensions settings, byte[] assembly)
         {
             var isValid = true;
+
+#if DEBUG
+            // Add Experimenal namespaces in Debug compile if it has not been added in allow list
+            if (!settings.AllowPowerFxNamespaces.Contains("Experimental"))
```

Comment:
Could it be better to use a class member const and avoid any possibility of errors as typing this string in multiple locations?

Reply:
Updated